concepts.md

```diff
@@ -187,15 +187,19 @@ of [process requirements](#Requirements_and_hints).
 The generic execution sequence of a CWL process (including workflows and
 command line line tools) is as follows.
 
-1. Load, process and validate a CWL document, yielding a process object.
-2. Load input object.
-3. Validate the input object against the `inputs` schema for the process.
-4. Validate process requirements are met.
-5. Perform any further setup required by the specific process type.
-6. Execute the process.
-7. Capture results of process execution into the output object.
-8. Validate the output object against the `outputs` schema for the process.
-9. Report the output object to the process caller.
+1. Load, process and validate a CWL document, yielding one or more process objects.
+1. Load input object.
+1. If there are multiple process objects (due to [`$graph`](SchemaSalad.html#Document_graph))
```

Comment:
Switch the ordering (choose `#main` before `Load input object`)

Reply:
Upon review, "Load input object" should be the very first, as it can specify which CWL document to load (and which Process is the entry point) via `cwl:tool`

Reply:
Yes, you're right.